en/docs/next/ai-workspace/genai-applications.md

```
@@ -72,56 +71,56 @@ The application is created within the current project and becomes the shared rep
-- Combine GenAI Applications with App LLM Proxies when an application also needs its own authentication, guardrails, or routing behavior.
+- Combine GenAI applications with App LLM proxies when an application also needs its own authentication, guardrails, or routing behavior.
 
 ## Related
 
-- [App LLM Proxies](llm-proxies/overview.md) - Create application-specific endpoints and controls
-- [Insights](insights.md) - Analyze usage by application
-- [Invoke via SDKs](using-sdks.md) - Use mapped keys when calling providers and proxies
\ No newline at end of file
+- [App LLM proxies overview](llm-proxies/overview.md): create application-specific endpoints and controls
+- [Insights](insights.md): analyze usage by application
+- [Invoke providers and proxies via SDKs](using-sdks.md): use mapped keys when calling providers and proxies
```

markdownlint-cli2 (0.23.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)